auditable-logging-app/Makefile

```diff
@@ -10,9 +10,17 @@ build: ## 🔨 Build the Logging Application
 	@echo -e "\e[34m$@\e[0m" || true
 	@npm run build
 
+build-virtual: build ## 📦 Build Virtual container image from Dockerfile
+	@echo -e "\e[34m$@\e[0m" || true
+	@../scripts/build_image.sh virtual
+
```

Comment:
we are missing `build-enclave` and `test-docker-enclave` in this app makefile. is this intended?

Reply:
Yes, because I can't test it! 😂